tools/snp.sh

```diff
@@ -974,7 +1103,11 @@ main() {
   
   # Set SETUP_WORKING_DIR for non-upm
   if ! $UPM; then
-    SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/non-upm"
+    if [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; then 
+          SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/non-upm"
+    else
+      echo "Present original AMDSEV repository has non-upm branch ...so skipping in RHEL"
+    fi
```

Comment:
The echo doesn't make sense here. Please read these statements and make sure they make sense before sticking them in the code.